packages/clerk-js/src/ui/components/OrganizationProfile/ActiveMembersList.tsx

```diff
@@ -32,22 +36,28 @@ export const ActiveMembersList = () => {
     return null;
   }
 
-  //TODO: calculate if user is the only admin
-  const canChangeOwnAdminRole = isAdmin && organization?.membersCount > 1;
-
   const handleRoleChange = (membership: OrganizationMembershipResource) => (newRole: MembershipRole) => {
     if (!isAdmin) {
       return;
     }
-    return card.runAsync(membership.update({ role: newRole })).catch(err => handleError(err, [], card.setError));
+    return card
+      .runAsync(async () => {
+        await membership.update({ role: newRole });
+        await (adminMembers as any).unstable__mutate?.();
+      })
+      .catch(err => handleError(err, [], card.setError));
   };
 
   const handleRemove = (membership: OrganizationMembershipResource) => () => {
     if (!isAdmin) {
       return;
     }
     return card
-      .runAsync(membership.destroy())
+      .runAsync(async () => {
+        const destroyedMembership = membership.destroy();
+        await (adminMembers as any).unstable__mutate?.();
```

Comment:
❓ Why do we have to cast `as any` here?

Reply:
unstable__mutate doesn't exist in the type.

This was a pattern that existed before. But I believe it makes sense to revisit this one ☝️

Reply:
Makes sense! Thanks for the context